### ISSUE TITLE
Can't open file (MacBook Air M1)

### ISSUE THREAD

**Issue by SeaDaddytheRed** [justdan96/tsMuxer#405]
Hello, I followed the dependency instructions in the readme but I am still unable to open any files.  I get the following error.

`dyld: Library not loaded: /usr/local/opt/freetype/lib/libfreetype.6.dylib
  Referenced from: /Volumes/WD500/nightly 2021-02-13/tsMuxerGUI.app/Contents/MacOS/tsmuxer
  Reason: image not found`

**Comment by justdan96** [comment]
Can you confirm your OS version and if the issue still exists with the latest versions?

**Comment by SeaDaddytheRed** [comment]
I'm on 11.4 and just tried you latest release.  Got the following error.

dyld: Library not loaded: /usr/local/opt/freetype/lib/libfreetype.6.dylib
  Referenced from: /private/var/folders/62/5wkppw817r1b739s9p2y5fyr0000gn/T/AppTranslocation/4F827057-0A1E-450E-BA4C-0E50FBFB0E0D/d/tsMuxerGUI.app/Contents/MacOS/tsmuxer
  Reason: image not found

**Comment by justdan96** [comment]
Might be related to #443

**Comment by justdan96** [comment]
Just out of curiosity, what error do you get if you try to open the tsMuxer executable from this artefact: https://github.com/justdan96/tsMuxer/actions/runs/1215042189 ?

**Comment by SeaDaddytheRed** [comment]
Sorry, but I don't know how to download from that link.  If I click txmuxer at the bottom it downloads a zip but once I open it I am not sure what to do with the file.  It's not an app file.

**Comment by justdan96** [comment]
I have borrowed an M1 Mac - the artefact is the tsMuxer binary executable and I wasn't expecting it to run.

If I check the file and its architecture:
```
% file tsMuxeR 
tsMuxeR: Mach-O 64-bit executable x86_64
```

This must mean we are both running the file with Rosetta 2:
https://www.macrumors.com/how-to/install-rosetta-m1-apple-silicon-mac/

```
 % ./tsMuxeR --version
tsMuxeR version git-8474c6b. github.com/justdan96/tsMuxer
```

It would be good to get a native executable but for now it appears Rosetta 2 is a decent workaround.

**Comment by francescomannarino** [comment]
I keep getting the same error on my Air M1, despite trying to use the last available nightly.
i also tried to replace the artifact i found in the thread, but it doesn't work in that case either.

dyld: Library not loaded: /opt/local/lib/libfreetype.6.dylib
  Referenced from: /private/var/folders/l9/mqhyycws6bd738k0b63ntrqm0000gn/T/AppTranslocation/C7CAE32E-8422-4C18-A43B-F74406FDA609/d/tsMuxerGUI.app/Contents/MacOS/tsmuxer
  Reason: image not found

**Comment by justdan96** [comment]
Should be solved after #544 is merged
